relayer/rly/cosmos_relayer.go

```diff
@@ -65,7 +65,12 @@ const (
 // to distinguish between multiple rly versions.
 func Capabilities() map[relayer.Capability]bool {
 	m := relayer.FullCapabilities()
+
+	// These two capabilities are not supported in the 2.0.0-beta4 tag,
+	// but they do work on the current main branch of relayer.
+	// The first 2.0 release candidate should be able to have full capabilities.
 	m[relayer.TimestampTimeout] = false
+	m[relayer.FlushAcknowledgements] = false
```

Comment:
Do we need a GH issue to remind us to adjust these lines when 2.0 released?

Reply:
It's front of mind for me, so I wasn't on adding an issue to track it.